### ISSUE TITLE
Relative dates is not translated

### ISSUE THREAD

**Issue by BlackSkorpio** [koken/issues#5]
When using `<koken:time relative="true" />` the output is not translatable.

**Steps to reproduce:**
Replace any instance of `<koken:time />` with  `<koken:time relative="true" />`

**Comment by MikeCostello** [comment]
consider updating [timeago](https://github.com/rmm5t/jquery-timeago) or add locales

**Comment by BlackSkorpio** [comment]
Maybe updating the core timeago and letting themes handle the locales?
Tested the core timeago: and that gives a strange output in swedish 

> För, en dag, sedan

Where it should be

> För en dag sedan

Loading the latest`jquery.timeago.js` corrects this.

Locales can easily be handled in the themes via 

```
<script>
    <koken:include file="inc/js/jquery.timeago.{{ settings.language }}.js" />
</scrip>`
```

**Comment by MikeCostello** [comment]
Fixed in v0.12.14 https://github.com/koken/koken/commit/bb639a41d3151188728a25e7da3ed9d9d097b008

No need to add locale based `koken:include` in theme, it's handled in Koken core.
